azkaban-common/src/main/java/azkaban/executor/ExecutableFlow.java

```diff
@@ -191,6 +202,10 @@ public void setSubmitTime(long submitTime) {
     this.submitTime = submitTime;
   }
 
+  public void setSlaOptions(List<SlaOption> slaOptions) {
+    this.slaOptions = slaOptions;
+  }
+
   @Override
   public Map<String, Object> toObject() {
```

Comment:
This entire construct should probably be refactored as well. There are standard libraries like `gson` and `jackson` to do json serde. This is simply reinventing the wheel. 

If we do that then the `toObject` and `fillExecutableFromMapObject` methods interfaces can be removed and along with it, all the constants etc can be cleaned up as well.